lib/matplotlib/artist.py

```diff
@@ -1437,7 +1437,12 @@ def setp(obj, *args, **kwargs):
       >>> setp(lines, linewidth=2, color='r')        # python style
     """
 
-    insp = ArtistInspector(obj)
```

Comment:
```
14:24 $ git grep ArtistInspector
CHANGELOG:           hardcopy and other use.  Ther ArtistInspector could use a
doc/users/whats_new.rst:    differently for hardcopy and other use.  The ArtistInspector
lib/matplotlib/artist.py:        return ArtistInspector(self).properties()
lib/matplotlib/artist.py:class ArtistInspector(object):
lib/matplotlib/artist.py:        the :class:`~matplotlib.artist.ArtistInspector`.
lib/matplotlib/artist.py:        insp = ArtistInspector(obj)
lib/matplotlib/artist.py:    insp = ArtistInspector(objs[0])
lib/matplotlib/artist.py:        return '\n'.join(ArtistInspector(a).pprint_setters_rest(
lib/matplotlib/artist.py:        return '\n'.join(ArtistInspector(a).pprint_setters(leadingspace=2))
```

It hopefully many of these use cases will be replaced by traitlet based methods.